api.go

```diff
@@ -108,36 +108,18 @@ func (h Header) normalizeKey(k string) string {
 //
 //   New(TCPDialer("127.0.0.1:783"))
 //   New(UnixDialer("/path/to/sock.unix"))
-//   New(BuildGenericDialer("127.0.0.1:783", "tcp", time.Second))
-//
 func New(d Dialer) *Client {
 	return &Client{dialer: d}
 }
 
-// BuildGenericDialer is a generic method to build a Dialer (cf `New` method)
-func BuildGenericDialer(addr string, proto string, timeout time.Duration) Dialer {
```

Comment:
@Carpetsmoker This was useful because it provided a way to specify the timeout. Without it the 20sec timeout is imposed.

Reply:
@Carpetsmoker any update on this PR? :)